fastlane/lanes/release.rb

```diff
@@ -157,18 +304,39 @@
   desc 'Trigger the final release build on CI'
   lane :finalize_release do |options|
     UI.user_error!('To finalize a hotfix, please use the finalize_hotfix_release lane instead') if ios_current_branch_is_hotfix
+    ensure_git_branch(branch: '^release/') # Match branch names that begin with `release/`
+    ensure_git_status_clean
+
+    message = "Finalizing release: #{current_release_version}\n"
+    if options[:skip_confirm]
+      UI.message(message)
+    else
+      UI.user_error!('Aborted by user request') unless UI.confirm("#{message}Do you want to continue?")
```

Comment:
Have you considered adding the `\n` in the `UI.confirm` only? `UI.message` should already handle separating messages in new lines.

```suggestion
    message = "Finalizing release: #{current_release_version}"
    if options[:skip_confirm]
      UI.message(message)
    else
      UI.user_error!('Aborted by user request') unless UI.confirm("#{message}\nDo you want to continue?")
```

Reply:
@mokagio I updated this in https://github.com/wordpress-mobile/WordPress-iOS/commit/75334b7c2ba858e1da781ee0d0f95c0ab7805d06. Instead of moving the `/n` to the `UI.confirm`, I changed it to always print the `message` and only leave the "Do you want to continue?" in the `UI.confirm`. That was based on some feedback from Oguz in WPAndroid: https://github.com/wordpress-mobile/WordPress-Android/pull/19288#discussion_r1346175218